src/App.js

```diff
@@ -1,14 +1,33 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+import "./app.css";
+
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      tasks: []
+    };
+  }
+  
+  addTodo = (task) => {
+    const newTask = {
+      id: Date.now(),
+      task: task,
+      completed: false
+    }
+    this.setState({tasks: [...this.state.tasks, newTask]});
```

Comment:
I wouldn't recommend spreading your tasks in due to random issues with the async actions of react rendering but it will only give you problems on extremely rare occasions.